unit/src/MFEMMaterialTest.C

```diff
@@ -223,6 +223,7 @@ TEST_F(MFEMMaterialTest, MFEMGenericConstantVectorMaterial_PW)
 
   mfem::VectorCoefficient & coef = _mfem_problem->getCoefficients().getVectorCoefficient("coef1");
   auto c = dynamic_cast<mfem::PWCoefficient *>(&coef);
+  EXPECT_NE(c, nullptr);
```

Comment:
This fails (c is nullptr). What type should this be? Found this with an unused variable warning.

Reply:
Changed to `PWVectorCoefficient`